treeherder/seta/job_priorities.py

```diff
@@ -87,7 +55,7 @@ def _gecko_decision_task_request(project):
             if task_request.counter % jp.priority != 0:
                 job_priorities.append(jp)
 
-    return _process(build_system='taskcluster', job_priorities=job_priorities)
+    return _process(project, build_system='taskcluster', job_priorities=job_priorities)
```

Comment:
Hi Ed,

I've been going around in circles trying to figure out how to test this. The only difference that the repo/project name makes in seta is when getting the list of runnable jobs for that repo; then the job priorities api will make sure any low value tasks exist in that list of runnable apis. There's no indication in the data returned from the job priorities seta api itself of which repo the jobs were confirmed against.

The only thing I can think of doing is adding a check in the code itself to verify that the 'repository' in the meta data returned by list_runnable_jobs matches the 'project' name (and if not raise an exception):

https://github.com/armenzg/treeherder/blob/1f44266864a789a6d1901553053329c1560d3ac0/treeherder/etl/seta.py#L179

Then test for that exception in test_job_priorities.py by using a different respository_name in the runnable_jobs_data in tests/seta/conftest.py (that doesn't match the project name sent into seta_job_scheduling).

However I don't know if that's valuable because the list_runnable_jobs method simply adds the received 'project' parameter to the returned data:

https://github.com/armenzg/treeherder/blob/1f44266864a789a6d1901553053329c1560d3ac0/treeherder/etl/runnable_jobs.py#L215

Any suggestions? Thanks!